dev_guide/templates.adoc

```diff
@@ -82,13 +121,27 @@ $ oc create -f <filename> -n <project>
 The template is now available to be selected for a configuration using the web
 console or the CLI.
 
-=== Generating a Configuration
-Generate a configuration with the following command:
+=== Generating a list of resources
+Generate a list of resources with the following command:
 
 ----
 $ oc process -f <filename>
 ----
 
+You can also process the uploaded template by calling the process command using:
+
+----
+$ oc process uploaded-template-name
+----
+
+The process command also takes a list of templates you can process to a list of
+resources  In that case, every template will be processed and the resulting list
+of resources will contain objects from all templates passed to a process command:
```

Comment:
Suggestion:

The process command also takes a list of templates you can process. 
In that case, every template will be processed and the resulting list
will contain objects from all templates passed to a process command: